app-web/src/components/PrimaryFilter/PrimaryFilter.js

```diff
@@ -0,0 +1,140 @@
+/*
```

Comment:
Only change made to this file was  it was converted from a `class Component` to `const Component`